addons/packages/pinniped/0.22.0/bundle/vendir.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+minimumRequiredVersion: 0.12.1
```

Comment:
This is still 0.12.1 as well.

Reply:
Fixed and pushed. Also ran `vendir sync`.